erpnext/education/doctype/quiz_question/quiz_question.json

```diff
@@ -20,7 +20,7 @@
   {
    "fetch_from": "question_link.question",
    "fieldname": "question",
-   "fieldtype": "Data",
+   "fieldtype": "Text Editor",
    "in_list_view": 1,
    "label": "Question",
    "read_only": 1
```

Comment:
The field over here is `read_only`. Questions are created in the Question doctype so the need for fieldtype to be Text Editor is over there. This field just fetches the content from there. So both of them need to be of type Text Editor. Also as I mentioned before, please change the modified timestamp otherwise these changes won't get synced.